modules/wms/src/services/ogc/wms-service.ts

```diff
@@ -234,8 +234,12 @@ export class WMSSource extends ImageSource<WMSSourceProps> {
   }
 
   async getImage(parameters: GetImageParameters): Promise<ImageType> {
-    // @ts-expect-error
-    return await this.getMap(parameters);
+    const {boundingBox, ...rest} = parameters;
```

Comment:
Nit: Add a comment to show what is going on...
```suggestion
    // Replace the GetImage `boundingBox` parameter with the WMS flat `bbox` parameter.
    const {boundingBox, ...rest} = parameters;
```